README.md

```diff
@@ -1479,6 +1479,93 @@ validateDatamobRequirements: ({phoneNumber: string, tokenPassword: string}) => P
 
 -   `requirements`: A map with the requirements.
 
+### displayQualtricsIntercept
+
+<kbd>App version >=24.12</kbd>
```

Comment:
All methods will be available only for iOS by the moment. In android it's expected to be integrated at some next version.
We can just a "Only for iOS" comment, and when it is integrated in android we can update the doc with the proper version.